internal/scripts/update-kclvm.sh

```diff
@@ -53,6 +53,15 @@ touch $kclvm_install_dir/bin/kclvm_cli
 rm $kclvm_install_dir/bin/kclvm_cli
 cp ./target/release/kclvm_cli $kclvm_install_dir/bin/kclvm_cli
 
+# build kcl LSP server
+
+cd $topdir/kclvm/tools/src/LSP
+cargo build --release
+
+touch $kclvm_install_dir/bin/kcl-language-server
+rm $kclvm_install_dir/bin/kcl-language-server
+cp $topdir/kclvm/target/release/kcl-language-server $kclvm_install_dir/bin/kcl-language-server
```

Comment:
The windows script here also needs to be updated with the kcl-language-server binary. Ref: KCLVM/scripts/build-windows/build.bat.